<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1312</width>
    <height>640</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1312</width>
    <height>640</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1312</width>
    <height>640</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Sistema de Visión Estereoscópica</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLabel" name="camaraIzquierda">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>640</width>
      <height>480</height>
     </rect>
    </property>
    <property name="maximumSize">
     <size>
      <width>640</width>
      <height>480</height>
     </size>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="camaraDerecha">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>30</y>
      <width>640</width>
      <height>480</height>
     </rect>
    </property>
    <property name="maximumSize">
     <size>
      <width>640</width>
      <height>480</height>
     </size>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>141</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>CAMARA IZQUIERDA</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>10</y>
      <width>141</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>CAMARA DERECHA</string>
    </property>
   </widget>
   <widget class="QPushButton" name="capturaEscena">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>520</y>
      <width>191</width>
      <height>51</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="text">
     <string>Capturar Escena</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>520</y>
      <width>191</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>FUNCIONES:</string>
    </property>
   </widget>
   <widget class="QPushButton" name="rectificarImagenes">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>550</y>
      <width>201</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Rectificar Imagenes</string>
    </property>
   </widget>
   <widget class="QPushButton" name="mapaDisparidad">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>590</y>
      <width>201</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Generar Mapa de Disparidad</string>
    </property>
   </widget>
   <widget class="QPushButton" name="distanciaObjetos">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>550</y>
      <width>201</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Distancias a Objetos</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>520</y>
      <width>161</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Calcular distancias</string>
    </property>
   </widget>
   <widget class="QPushButton" name="distanciaNavegacion">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>590</y>
      <width>201</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Distancias Navegacion</string>
    </property>
   </widget>
   <widget class="QPushButton" name="abrirDirectorio">
    <property name="geometry">
     <rect>
      <x>1100</x>
      <y>590</y>
      <width>191</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Abrir carpeta de archivos</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>520</y>
      <width>451</width>
      <height>111</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:12pt; font-weight:600; text-decoration: underline;&quot;&gt;Procedimiento Sistema de Visión Estereoscópica:&lt;/span&gt;&lt;br/&gt;&lt;span style=&quot; font-style:italic;&quot;&gt;Alternativa Calibrar y rectificar cámara (stereocalib.yml)&lt;/span&gt;&lt;br/&gt;&lt;span style=&quot; font-style:italic; color:#315389;&quot;&gt;1° Obtener par estereo (Capturar Escena)&lt;br/&gt;2° Rectificar imágenes&lt;br/&gt;3° Obtener mapa de disparidad (mapaDisparidad.png)&lt;br/&gt;4° Calculo de distancias: Objetos (distancias.yml) o Para navegación&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>1110</x>
      <y>560</y>
      <width>181</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Ver directorio de archivos:</string>
    </property>
   </widget>
   <widget class="QPushButton" name="calibrarCAM">
    <property name="geometry">
     <rect>
      <x>1100</x>
      <y>525</y>
      <width>191</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Calibrar y Rectifica CAM</string>
    </property>
   </widget>
   <widget class="QLabel" name="cantidadCapTab">
    <property name="geometry">
     <rect>
      <x>136</x>
      <y>596</y>
      <width>61</width>
      <height>31</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>580</y>
      <width>61</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:10pt;&quot;&gt;Cantidad&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QPushButton" name="capturaTablero">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>580</y>
      <width>121</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Captura Tablero</string>
    </property>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
